src/factories/plugin-factory.ts

```diff
@@ -38,6 +39,7 @@ export interface PluginFactoryOptions {
   targetBranch: string;
   repositoryConfig: RepositoryConfig;
   manifestPath: string;
+  separatePullRequests: boolean;
```

Comment:
This should probably be optional and have a default value. The ability to register plugins and pluginFactories is part of the public API (`PluginFactoryOptions`). It's unlikely that someone is providing their own `PluginFactoryOptions` instance, but a default wouldn't hurt.